routes/inventory-routes.js

```diff
@@ -0,0 +1,85 @@
+const db = require("../models");
+
+module.exports = function(app){
+
+    // GET full inventory
+    app.get("/api/inventory", function(req,res){
+
+        db.Inventory.findAll({
+            include: [db.Products][db.Seller]
+        }).then(function(dbInventory){
+            res.json(dbInventory);
+        });
+    });
+
+    // GET inventory by seller
+    app.get("/api/inventory/seller/:id", function(req,res){
+
+        db.Inventory.findAll({
+            where: {
+                sellerId: req.params.id
+            },
+            include: [db.Products][db.Seller]
```

Comment:
does this work?  if it doesn't maybe try [db.Products, db.Seller], if it does work, let us know how.

Reply:
[db.Products][db.Seller] does not work. I switched to your recommendation of [db.Products, db.Seller] and it works. However, I may not need this line of code anymore. I added include: [db.Products] to the seller-routes. Now when I load a seller, it list all the products they are selling plus stock.